src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -989,4 +1090,92 @@ RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1,
 #endif // backend
 }
 
+/******************** glitch ********************/
+
+RppStatus rppt_glitch_gpu(RppPtr_t srcPtr,
+                          RpptDescPtr srcDescPtr,
+                          RppPtr_t dstPtr,
+                          RpptDescPtr dstDescPtr,
+                          Rpp32u *xOffsetR,
+                          Rpp32u *yOffsetR,
+                          Rpp32u *xOffsetG,
+                          Rpp32u *yOffsetG,
+                          Rpp32u *xOffsetB,
+                          Rpp32u *yOffsetB,
+                          RpptROIPtr roiTensorPtrSrc,
+                          RpptRoiType roiType,
+                          rppHandle_t rppHandle)
+{
```

Comment:
copy_param_uint calls should be present for all the parameters for copy the data from HOST to HIP memory
xOffsetR, xOffsetG, xOffsetB
yOffsetR, yOffsetG, yOffsetB

Outputs won't be correct without these changes. Please check them

Reply:
Done